onnxruntime/core/session/inference_session.h

```diff
@@ -425,5 +434,7 @@ class InferenceSession {
   //CustomRegistry objects own the corresponding KernelRegistry and OnnxRuntimeOpSchemaRegistry objects.
   //So its lifetime should be same as its constituents. This vector is to extend the lifetime of the owner.
   std::vector<std::shared_ptr<CustomRegistry>> custom_registries_;
+  std::vector<std::unique_ptr<PyCustomOp>> pyops_;
```

Comment:
Let's define a CustomInteropOp struct containing the language ops and pass a ref of it to the LoadInteropOp function to fill it in. Also, surround it in an ifdef.

Reply:
Done.